.github/workflows/test.yml

```diff
@@ -21,6 +21,64 @@ jobs:
           java-version: 17
           distribution: 'temurin'
           cache: 'maven'
+      # look for dependencies in maven
+      - name: maven-settings-xml-action
+        uses: whelk-io/maven-settings-xml-action@v21
+        with:
+          repositories: |
+            [
+              {
+                "id": "liquibase",
+                "url": "https://maven.pkg.github.com/liquibase/liquibase",
+                "releases": {
+                  "enabled": "false"
+                },
+                "snapshots": {
+                  "enabled": "true",
+                  "updatePolicy": "always"
+                }
+              },
+              {
+                "id": "liquibase-pro",
+                "url": "https://maven.pkg.github.com/liquibase/liquibase-pro",
+                "releases": {
+                  "enabled": "false"
+                },
+                "snapshots": {
+                  "enabled": "true",
+                  "updatePolicy": "always"
+                }
+              },
+                {
+                  "id": "liquibase-super-pom",
```

Comment:
Is it the correct name? Is it "super"? For id and url.

Reply:
Yeah sorry, I deleted all references to it since it is not needed anymore